hq/app/aws/s3/S3.scala

```diff
@@ -19,7 +19,6 @@ object S3 {
           "unable to get S3 object for the unrecognised user job",
           "I haven't been able to get the S3 object for the unrecognised user job, which contains the Janus data",
           500,
-          context = Some(e.getMessage),
```

Comment:
This is my mistake. I can't think why I would want to remove context, but keep throwable! Sorry about this! I'll note to add this back in another PR.

Reply:
No, this shouldn't be here and is something I removed early on in the Remediation branch work. `context` has a special meaning in Failure, in that it can tell you the location that caused the error.

For example,
If we're taking a form submission from the user and they've given an incorrect field, the context can be used to identify which field was broken. This is important if we need to line up error message on the client. If the submission is invalid and we get several errors, the context lets us put the error message next to the right input box.
It's not just for UIs though. As another example, if we're trying to deserialise a JSON string, the context of a failure might tell you that the problem was in the `date` field.

The exception's message doesn't give "context" in this sense. It's just the description of the failure, and that should be in the message itself, or included in the log entry directly as part of the (optional) `throwable` field.

Let's chat about it a little more, but no need to restore this.

Reply:
Makes sense. Thanks @adamnfish! 👍🏽